frontend/packages/kubevirt-plugin/integration-tests/tests/vm.actions.scenario.ts

```diff
@@ -0,0 +1,334 @@
+    const vmName = `vm-list-actions-${testName}`;
+
+    beforeAll(async () => {
+      testVm.metadata.name = vmName;
+      createResource(testVm);
+      addLeakableResource(leakedResources, testVm);
+
+      // Navigate to Virtual Machines page
+      await browser.get(`${appHost}/k8s/ns/${testName}/virtualmachines`);
+      await isLoaded();
+      await fillInput(textFilter, vmName);
+      await resourceRowsPresent();
+    });
+
+    it(
+      'Starts VM',
+      async () => {
+        await listViewAction(vmName)('Start', true);
+        await fillInput(textFilter, vmName);
+        await browser.wait(
+          until.presenceOf(statusIcon(statusIcons.running)),
+          VM_BOOTUP_TIMEOUT_SECS,
+        );
+      },
+      VM_BOOTUP_TIMEOUT_SECS,
+    );
+
+    it(
+      'Restarts VM',
+      async () => {
+        await listViewAction(vmName)('Restart', true);
+        await fillInput(textFilter, vmName);
+        await browser.wait(
+          until.or(
+            until.presenceOf(statusIcon(statusIcons.starting)),
+            until.presenceOf(statusIcon(statusIcons.error)),
+          ),
+        );
+        await browser.wait(
+          until.presenceOf(statusIcon(statusIcons.running)),
+          VM_BOOTUP_TIMEOUT_SECS,
+        );
+      },
+      VM_ACTIONS_TIMEOUT_SECS,
+    );
+
+    it('Stops VM', async () => {
+      await listViewAction(vmName)('Stop', true);
+      await fillInput(textFilter, vmName);
+      await browser.wait(until.presenceOf(statusIcon(statusIcons.off)), VM_STOP_TIMEOUT_SECS);
+    });
+
+    it('Deletes VM', async () => {
+      await listViewAction(vmName)('Delete', true);
+      await isLoaded();
+      await fillInput(textFilter, vmName);
+      await browser.wait(until.and(waitForCount(resourceRows, 0)), PAGE_LOAD_TIMEOUT_SECS);
+      removeLeakableResource(leakedResources, testVm);
+    });
+  });
+
+  describe('Test VM detail view actions dropdown', () => {
+    const vmName = `vm-detail-actions-${testName}`;
+    const vm = new VirtualMachine({ name: vmName, namespace: testName });
+
+    beforeAll(async () => {
+      testVm.metadata.name = vmName;
+      createResource(testVm);
+      addLeakableResource(leakedResources, testVm);
+      await vm.navigateToTab(TABS.OVERVIEW);
+    });
+
+    it(
+      'Starts VM',
+      async () => {
+        await vm.action('Start');
+      },
+      VM_BOOTUP_TIMEOUT_SECS,
+    );
+
+    it(
+      'Restarts VM',
+      async () => {
+        await vm.action('Restart');
+      },
+      VM_ACTIONS_TIMEOUT_SECS,
+    );
+
+    it('Stops VM', async () => {
+      await vm.action('Stop');
+    });
+
+    it('Deletes VM', async () => {
+      await vm.action('Delete');
+      await isLoaded();
+      await fillInput(textFilter, vmName);
+      await browser.wait(until.and(waitForCount(resourceRows, 0)), PAGE_LOAD_TIMEOUT_SECS);
+      removeLeakableResource(leakedResources, testVm);
+    });
+  });
+});
+
+describe('Test VM Migration', () => {
```

Comment:
This should go either under `Test VM actions` scenario or be moved in a separate file (which is preferred, imo).

Reply:
Done.